setup.py

```diff
@@ -10,20 +10,21 @@
 setup(
     name='django-walletpass',
     python_requires='>=3.10.0',
-    version='4.1',
+    version='4.2',
     author='Develatio Technologies S.L.',
     author_email='contacto@develat.io',
     packages=find_packages(),
     include_package_data=True,
     url='http://github.com/develatio/django-walletpass/',
     license='BSD',
     install_requires=[
-        'Django>=3.2.9',
+        'Django>=3.2.9,<=5.1',
```

Comment:
why do we want <=5.1?

Reply:
because this is, what we have under the test matrix running and we can ensure that it will truly work. if we leave it open, it can be installed with newer Django versions, but we cannot guarantee that it will work.

Reply:
🤔 I'd rather do it like any other Django library (take for example DRF) and not limit the newer versions. Doing so doesn't force us to push a new version just because a new version of Django was released.